base/shell.jl

```diff
@@ -158,30 +166,30 @@ end
 
 function print_shell_escaped(
         io::IO, cmd::AbstractString, args::AbstractString...;
-        special::AbstractString=shell_special
+        special::AbstractString=""
     )
     print_shell_word(io, cmd, special)
     for arg in args
         print(io, ' ')
         print_shell_word(io, arg, special)
     end
 end
-print_shell_escaped(io::IO; special::String=shell_special) = nothing
+print_shell_escaped(io::IO; special::String="") = nothing
 
 """
-    shell_escape(args::Union{Cmd,AbstractString...}; special::AbstractString="$shell_special")
+    shell_escape(args::Union{Cmd,AbstractString...}; special::AbstractString="")
 
 The unexported `shell_escape` function is the inverse of the unexported `shell_split` function:
 it takes a string or command object and escapes any special characters in such a way that calling
 `shell_split` on it would give back the array of words in the original command. The `special`
 keyword argument controls what characters in addition to whitespace, backslashes, quotes and
-dollar signs are considered to be special. Examples:
+dollar signs are considered to be special (default: none). Examples:
+
+    julia> Base.shell_escape("cat", "/foo/bar baz", "&&", "echo", "done")
+    "cat '/foo/bar baz' && echo done"
 
     julia> Base.shell_escape("echo", "this", "&&", "that")
     "echo this '&&' that"
```

Comment:
At the cost of both 0.5.2 and 0.6.0-rc1 coming out later because I'd be spending time fixing this instead, sure.